docs/dockerhub.md

```diff
@@ -0,0 +1,160 @@
+# jpo-conflictmonitor
+
+## GitHub Repository Link
+https://github.com/usdot-jpo-ode/jpo-conflictmonitor
+
+## Purpose
+The JPO Conflict Monitor is a real time validation system to verify corresponding SPAT, MAP and BSM messages.
+
+## How to pull the latest image
+The latest image can be pulled using the following command:
+> docker pull usdotjpoode/jpo-conflictmonitor:develop
+
+## Required environment variables
+	• DOCKER_HOST_IP
+	• DB_HOST_IP
+	• CONNECT_BOOTSTRAP_SERVERS
+
+## Direct Dependencies
+	- MongoDB
+	- Kafka Connect
+
+## Indirect Dependencies
+The Conflict Monitor will not receive messages to process if the ODE is not running.
```

Comment:
Not just the ODE, the jpo-geojsonconverter is also required since it only consumes processed MAP and SPaT data with geoJSON. It consume BSM traffic directly from the ODE, though.

Reply:
Modified this section to include the Geojson Converter as well